.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "starlette-mcp-server-quickstart",
+    "image": "mcr.microsoft.com/devcontainers/python:3.13-bookworm",
+    "forwardPorts": [8000],
```

Comment:
Does mcp dev inspector work inside the dev container? I remember trying it in Codespaces in the live stream and it failing presumably due to CORS issues.

Reply:
Tbh I didn't test it, just added logical configuration and was busy trying to get uv to work let me test it

Reply:
It works 🎉 
I think you opened the wrong port
<img width="1349" alt="image" src="https://github.com/user-attachments/assets/62a241be-1017-420c-bd42-a6b65c959ae7" />